.circleci/config.yml

```diff
@@ -416,6 +446,11 @@ jobs:
 
       - <<: *abort_job_if_only_docs_changes
 
+      - attach_workspace:
+          at: "<< pipeline.parameters.workspace-with-cli-binary-and-images-mountpoint >>"
+
+      - run: cp -R << pipeline.parameters.workspace-with-cli-binary-and-images-mountpoint >>/webapp engine/server/webapp
```

Comment:
@Dartoxian does this not happen as part of the `postbuild` step in `enclave-manager/scripts/build.sh` => `yarn build`?

Reply:
this step is what copies the build output between jobs:
* First the `build_enclave_manager_webapp` job will run, and save the build output into the circle ci workspace
* Second the `build_engine_server` job will run, this job reattaches the workspace and copies the build output into place  (this is what is happening here) before running it's own build steps

I chose to have a job for building emui separate from `build_engine_server` because the details of building (or testing) the emui are not relevant to the engine server, only the output

Reply:
Cool. I _think_ this should work

Reply:
:pray: